tests/cisco_ios/show_ip_bgp_neighbor/cisco_ios_show_ip_bgp_neighbor.raw

```diff
@@ -0,0 +1,97 @@
+BGP neighbor is 20.30.255.14,  remote AS 1255, internal link
```

Comment:
Do you possibly have output with two BGP neighbors that we can use for tests? This just ensures we can properly parse the output with multiple neighbors. If not, you can always duplicate the neighbors, but change some of the information as well.

Reply:
Yup! Added a new test with a bunch of neighbors in the most recent commit.